advntr/__main__.py

```diff
@@ -58,6 +58,8 @@ def main():
     outfmt_choices = ['text', 'bed', 'vcf']
     genotype_io_group.add_argument('-of', '--outfmt', metavar='<format>', default='text', choices=outfmt_choices,
                                    help='output format. Allowed values are {'+', '.join(outfmt_choices)+'} [%(default)s]')
+    genotype_io_group.add_argument('--disable_logging', action='store_true',
+                                   help='set this flag to stop writing to log file except for critical errors.')
```

Comment:
By default, it will be set to `False`? Maybe it's good to explicitly say what the default value is.

Reply:
It's by default set to `False`. But I put the default value to make it clear. Thanks for the comment.